integration/storagetest/maptests.go

```diff
@@ -193,3 +196,64 @@ func (*mapTests) TestTileRoundTrip(ctx context.Context, t *testing.T, ms storage
 		t.Fatalf("ReadWriteTransaction(): %v", err)
 	}
 }
+
+func (*mapTests) TestGetSetRoundTrip(ctx context.Context, t *testing.T, ms storage.MapStorage, as storage.AdminStorage) {
+	tree := mustCreateTree(ctx, t, as, storageto.MapTree)
+	mustSignAndStoreMapRoot(ctx, t, ms, tree, &types.MapRootV1{Revision: uint64(0)})
+
+	num := int64(10)
```

Comment:
Just inline this? I'm not sure the documentation provided by `num` materially decreases the magic number factor here.

Reply:
done